pmlc/conversion/tests/stripe_affine_roundtrip/load-store.mlir

```diff
@@ -16,3 +17,21 @@ attributes {stripe_attrs = {program = unit}} {
 // AFFINE-LABEL: func @load_constant_addr(
 // AFFINE-SAME: %[[BASE:.*]]: memref
 // AFFINE: affine.load %[[BASE]][128]
+
+// -----
+
+!fp32_1 = type !stripe<"tensor_ref !eltwise.fp32:1">
+func @store_constant_addr(%arg0: !fp32_1 {stripe.layout = !stripe<"tensor !eltwise.fp32(addr[2048:1])">}, %arg1: !eltwise.fp32)
+attributes {stripe_attrs = {program = unit}} {
+  stripe.parallel_for () {
+    %c0 = stripe.affine_poly () [], 128
+    %r0 = stripe.refine %arg0 (%c0) : !fp32_1
+    "stripe.store"(%r0, %arg1) : (!fp32_1, !eltwise.fp32) -> ()
```

Comment:
Still valid, but I re-enabled the compact form, `stripe.store %r0, %arg1 : !fp32_1` for the above, so you might want to go back to that.

Reply:
Maybe on the next commit to not induce a new CI run for a tiny fix.